### PR TITLE
Closes overlay when new token is selected

### PR DESCRIPTION
**Pull request by taleldayekh**
**Thank you for submitting this pull request :)**

Fixes # 

**Short description**
Previously the token overlay was closed only if the selected token was the same as the current token. Because of this, the user had to click a token twice when selecting a new token to close the overlay.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [x] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Click to open the token select overlay
2. Select any token other than the currently selected token

### CODE REVIEW

**Comment by kelsos** [comment]
@taleldayekh can you also link the issue it closes on `Fixes`?

**Comment by taleldayekh** [comment]
> can you also link the issue it closes on `Fixes`? and maybe changelog entry?

Sure, I'll do both.

File: raiden-dapp/src/components/TokenOverlay.vue [hunk 1]
```diff
@@ -127,6 +127,7 @@ export default class TokenOverlay extends Mixins(
     if (token === tokenAddress) {
       this.cancel();
     } else {
+      this.cancel();
```
Comment (nephix) [inline raiden-dapp/src/components/TokenOverlay.vue:130]: If `cancel()` needs to be called in every situation we can just pull it ouside and change the if to `token !== tokenAddress` to call `this.navigateToSelectTransferTarget(tokenAddress)`

Comment (kelsos) [reply raiden-dapp/src/components/TokenOverlay.vue:130]: Really nice suggestion :)

Comment (taleldayekh) [reply raiden-dapp/src/components/TokenOverlay.vue:130]: Alrighty, will do!

**Pull request merged**